src/common/Pair.ts

```diff
@@ -130,8 +186,8 @@ export default class Pair {
 
     if (![PAIR_ASK, PAIR_BID].includes(type)) throw new Error(`CreateOrderError: Wrong order type: ${type}`)
 
-    const base = { currency: BASE, amount: amount.times(price) }
-    const main = { currency: MAIN, amount }
+    const base = { currency: BASE, amount: amount }
+    const main = { currency: MAIN, amount: amount.div(price) }
```

Comment:
Implementation differences:
- https://github.com/swaponline/MultiCurrencyWallet/blob/master/src/bot/microbot/Pair.ts#L115
-https://github.com/swaponline/MultiCurrencyWallet/blob/master/src/core/swap.orders/Pair.ts#L133
-https://github.com/swaponline/MultiCurrencyWallet/blob/master/src/core/swap.orders/Pair.ts#L133
I think the `core` is correct one